src/routes/docs/configuring/custom-themes/+page.md

```diff
@@ -2,28 +2,33 @@
 title: Custom themes in Files
 ---
 
-Files comes with a large collection of custom background colors but additional customization is available for advanced users.
+Files has a number of options for customizing the theme.
 
-## Advanced themeing
+### Backdrop Material
+The backdrop is set to Mica Alt by default, but it can be changed to Acrylic from Settings > Appearance > Backdrop Material.
 
-To access theme values that are not exposed in the settings modal
+### Custom Fonts
+Files has built-in support for custom fonts, to apply a custom font start by opening the settings file from Settings > Advanced > Edit settings file.
+1. Close the Files app before continuing.
+2. Edit the value for `AppThemeFontFamily`
 
-1. Open Files settings > Advanced > Edit settings file. This will open the main settings file in your default text editor.
-2. Close the app before making any changes to the settings file.
+### Advanced themes
 
-### Available resources for themeing
+1. Open the settings file from Settings > Advanced > Edit settings file.
+2. Close the Files app before continuing.
+3. Edit these values as desired
 
 | Key                                 | Default value       |
 | ----------------------------------- | ------------------- |
 | `AppThemeBackgroundColor`           | `#00000000`         |
 | `AppThemeAddressBarBackgroundColor` | `#00000000`         |
 | `AppThemeSidebarBackgroundColor`    | `#00000000`         |
 | `AppThemeFileAreaBackgroundColor`   | `#00000000`         |
-| `AppThemeFontFamily`                | `Segoe UI Variable` |
 
-## Sample themes
+### Popular themes
+We put together some popular color combinations to help users create custom themes.
 
-### Glass (requires Windows 11)
+**Glass (requires Windows 11)**
```

Comment:
Using bold styling instead of headings here makes the text less accessible, so you should consider using a list or headings.

Reply:
This is the only way to make it stand out, we can't use headings because it's part of the "Advanced themes" section.